specification/sqlvirtualmachine/resource-manager/Microsoft.SqlVirtualMachine/preview/2021-11-01-preview/sqlvm.json

```diff
@@ -0,0 +1,2243 @@
+          "description": "Duration of the time window of a given day during which full backups can take place. 1-23 hours.",
+          "type": "integer"
+        },
+        "logBackupFrequency": {
+          "format": "int32",
+          "description": "Frequency of log backups. 5-60 minutes.",
+          "type": "integer"
+        }
+      }
+    },
+    "KeyVaultCredentialSettings": {
+      "description": "Configure your SQL virtual machine to be able to connect to the Azure Key Vault service.",
+      "type": "object",
+      "properties": {
+        "enable": {
+          "description": "Enable or disable key vault credential setting.",
+          "type": "boolean"
+        },
+        "credentialName": {
+          "description": "Credential name.",
+          "type": "string"
+        },
+        "azureKeyVaultUrl": {
+          "description": "Azure Key Vault url.",
+          "type": "string"
+        },
+        "servicePrincipalName": {
+          "description": "Service principal name to access key vault.",
+          "type": "string"
+        },
+        "servicePrincipalSecret": {
+          "description": "Service principal name secret to access key vault.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "update"
+          ]
+        }
+      }
+    },
+    "ServerConfigurationsManagementSettings": {
+      "description": "Set the connectivity, storage and workload settings.",
+      "type": "object",
+      "properties": {
+        "sqlConnectivityUpdateSettings": {
+          "$ref": "#/definitions/SqlConnectivityUpdateSettings",
+          "description": "SQL connectivity type settings."
+        },
+        "sqlWorkloadTypeUpdateSettings": {
+          "$ref": "#/definitions/SqlWorkloadTypeUpdateSettings",
+          "description": "SQL workload type settings."
+        },
+        "sqlStorageUpdateSettings": {
+          "$ref": "#/definitions/SqlStorageUpdateSettings",
+          "description": "SQL storage update settings."
+        },
+        "additionalFeaturesServerConfigurations": {
+          "$ref": "#/definitions/AdditionalFeaturesServerConfigurations",
+          "description": "Additional SQL feature settings."
+        },
+        "sqlInstanceSettings": {
+          "$ref": "#/definitions/SQLInstanceSettings",
+          "description": "SQL Instance settings."
+        }
+      }
+    },
+    "StorageConfigurationSettings": {
+      "description": "Storage Configurations for SQL Data, Log and TempDb.",
+      "type": "object",
+      "properties": {
+        "sqlDataSettings": {
+          "$ref": "#/definitions/SQLStorageSettings",
+          "description": "SQL Server Data Storage Settings."
+        },
+        "sqlLogSettings": {
+          "$ref": "#/definitions/SQLStorageSettings",
+          "description": "SQL Server Log Storage Settings."
+        },
+        "sqlTempDbSettings": {
+          "$ref": "#/definitions/SQLTempDbSettings",
```

Comment:
I also see this type has been changed from `SQLStorageSettings` to `SQLTempDbSettings`, **please double confirm.**
I believe this is inevitable therefore I am OK with this breaking change, but I wonder why our cross version breaking change pipeline is not reporting this.

Reply:
I've confirmed we are updating this type to `SQLTempDbSettings` as part of this api version. `SQLTempDbSettings` is an extended type of `SQLStorageSettings`